packages/components/src/navigation/back-button/index.tsx

```diff
@@ -26,9 +34,12 @@ function NavigationBackButton(
 		className
 	);
 
-	const parentMenuTitle = navigationTree.getMenu( parentMenu )?.title;
+	const parentMenuTitle =
+		parentMenu !== undefined
+			? navigationTree.getMenu( parentMenu )?.title
+			: undefined;
```

Comment:
⚠️  Runtime change:

Since the `getMenu` function doesn't accept `undefined` arguments, I just made the check more explicit here